doc/compiled.json

```diff
@@ -8775,6 +8775,24 @@
               "type": "string"
             }
           },
+          {
+            "description": "Download all translation keys, and remove the specified prefix where possible. Warning: this may create duplicate key names if other keys share the same name after the prefix is removed.",
+            "example": "prefix_",
+            "name": "translation_key_prefix",
+            "in": "query",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "description": "Only download translation keys containing the specified prefix, and remove the prefix from the generated file.",
+            "example": null,
+            "name": "use_translation_key_prefix_as_filter",
```

Comment:
This still seems like a really long name to me that's hard to remember. Maybe `filter_by_prefix` instead?

Reply:
I'm also still not sure it's good to have this option to disable it. It also violates our principle that imported files can be uploaded as is again, when there's a conflict. And also there might be formats where this is causing issues during rendering

Reply:
I can certainly change the name of the parameter ^^

> I'm also still not sure it's good to have this option to disable it. It also violates our principle that imported files can be uploaded as is again, when there's a conflict. And also there might be formats where this is causing issues during rendering
Do you suggest to by default use the prefix as a filter? In that case, how would you name a second parameter if the filtering is not wanted?
Regarding the issues: Maybe it's enough to warn the users. Let me ask our PM.

Reply:
`use_translation_key_prefix_as_filter` was renamed to `filter_by_prefix` 👍 
See also here: https://github.com/phrase/phrase/pull/13996